deny.toml

```diff
@@ -88,6 +88,7 @@ allow = [
   "BSL-1.0",
   "Unicode-3.0",
   "MPL-2.0",
+  "MIT-0",
```

Comment:
Ci failed because we depend on https://github.com/yescallop/borrow-or-share indirectly, @Boshen can you help to check if this license is safe to allow?

Reply:
> The "MIT No Attribution" or "MIT-0" license is a modification to the usual MIT license that removes the requirement for attribution. It is roughly the same as an MIT license consisting of a sandwich of (paragraph, line, paragraph), but it is missing the middle line of the 'sandwich'.

Repo: https://github.com/aws/mit-0